example/src/main/java/com/transferzero/test/Application.java

```diff
@@ -158,6 +158,26 @@ public static void getTransactionByExternalId(ApiClient apiClient) throws Except
         }
     }
 
+    public static void apiInterceptorExample(ApiClient apiClient) throws ApiException {
+
+        class ApiInterceptorImpl implements ApiClient.ApiInterceptor {
+            public Request handlePreRequest(Request request) {
+                System.out.println(request.body());
+                return request;
+            }
+
+            public Response handlePostResponse(Response response) {
+                System.out.println(response.body());
+                return response;
+            }
+        }
+
+        ApiInterceptorImpl ApiIntImpl = new ApiInterceptorImpl();
+        apiClient.setApiInterceptor(ApiIntImpl);
+        SendersApi api = new SendersApi(apiClient);
+        api.getSenders().externalId("1234567890").execute();
```

Comment:
Can you use the same `externalId` that is set on the Sender elsewhere in this file, rather than a random string that won't return anything? In theory people should be able to run this and get the same Sender back that they created in previous examples.